db/seeds.rb

```diff
@@ -143,16 +143,14 @@ def make_subcategories
 end
 
 def make_booklists
-  1.upto(5) do |n|
-    b = BookList.create! name: "booklist#{n}",
-                         published: false
-    b.users << User.find(n)
-  end
-  1.upto(5) do |n|
-    rand = Random.rand(Book.count - 3) + 1
-    rand.upto(rand+3) do |i|
-      BookList.find(n).books << Book.find(i)
+  0.upto(4) do |i|
+    b = BookList.new name: "booklist#{i}",
+                     published: false
+    b.users << User.find(i)
+    1.upto(5) do |j|
```

Comment:
Unused block argument - `j`. You can omit the argument if you don't care about it.